src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const BASE_PRICE = days * 40;
+
+  if (days < SHORT_TERM) {
+    return BASE_PRICE;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
```

Comment:
Good comment, apply the suggested solution:
![image](https://github.com/user-attachments/assets/f42478a1-2f59-4a66-8a56-3ca7bbfbfd96)